src/components/policy/test/policy_manager_impl_test.cc

```diff
@@ -86,7 +89,44 @@ class PolicyManagerImplTest : public ::testing::Test {
     delete listener;
   }
 
-  ::testing::AssertionResult IsValid(const policy_table::Table &table) {
+  ::testing::AssertionResult IsValid(const policy_table::Table& table) {
+    if (table.is_valid()) {
+      return ::testing::AssertionSuccess();
+    } else {
+      ::rpc::ValidationReport report(" - table");
+      table.ReportErrors(&report);
+      return ::testing::AssertionFailure() << ::rpc::PrettyFormat(report);
+    }
+  }
+};
+
+class PolicyManagerImplTest2 : public ::testing::Test {
+ protected:
+  PolicyManagerImpl* manager;
+  MockCacheManagerInterface* cache_manager;
+  MockUpdateStatusManager update_manager;
+  MockPolicyListener* listener;
+
+  void CreateLocalPT(const std::string& filename) {
+    // what to do here ?
```

Comment:
@orhan-mehmedov Please remove this comment. Please TODO or implement  this method.